compiler/luci/pass/src/FuseBCQPass.cpp

```diff
@@ -48,6 +58,11 @@ const std::string node_name_prefix(luci::NodeName node_name)
     const auto index = prefix.find("kernel/");
     prefix = prefix.substr(0, index - 1);
   }
+  else if (prefix.find("/bcqinfo_") != std::string::npos)
+  {
+    const auto index = prefix.find("/bcqinfo_");
```

Comment:
same here

Reply:
for `"Tensordot/"` we can change it in another PR if you accept

Reply:
Then, I will change all the things in another PR for consistency :)